source/usage-examples/updateMany.txt

```diff
@@ -8,14 +8,14 @@ Update Multiple Documents
    If you specify a callback method, ``updateMany()`` returns nothing. If you
    do not specify one, this method returns a ``Promise`` that resolves to the
    result object when it completes. See our guide on :doc:`Promises and
-   Callbacks </fundamentals/promises>` for more information, or the
+   Callbacks </fundamentals/promises>` for more information, or see the
    :node-api-4.0:`API documentation </classes/collection.html#~resultcallback>` for
    information on the result object.
 
 You can update multiple documents using the
-:node-api-4.0:`updateMany() </classes/collection.html#updatemany>` method.
-``updateMany()`` accepts a filter object and update document, and
-replaces documents in a collection that match the filter with the update
+:node-api-4.0:`collection.updateMany() </classes/collection.html#updatemany>` method.
+The ``updateMany()`` method accepts a filter object and update document and
+then replaces documents in a collection that match the filter with the update
```

Comment:
```suggestion
replaces documents in a collection that match the filter with the update
```
Suggestion: I don't think "then" should be in the sentence. With it, it sounds like "The method does `a` and then does` b`". But `b` is what happens with what is provided in `a`. So to me, it should sound like "The method does `a` and does` b`"

Reply:
I thought it might be clunky to have so many "and"s in that sentence. I may just copy the format over from the updateOne() page.